Model/Sales/TrackTraceHolder.php

```diff
@@ -111,45 +120,53 @@ public function createTrackTraceFromShipment(Order\Shipment &$shipment)
      */
     public function convertDataFromMagentoToApi($magentoTrack, $options)
     {
-        $address = $magentoTrack->getShipment()->getShippingAddress();
+        $this->consignment = ConsignmentFactory::createByCarrierId(BpostConsignment::CARRIER_ID);
+
+        $address      = $magentoTrack->getShipment()->getShippingAddress();
         $checkoutData = $magentoTrack->getShipment()->getOrder()->getData('delivery_options');
-        $deliveryType = $this->getDeliveryTypeFromCheckout($checkoutData);
+        $deliveryType = $this->consignment->getDeliveryTypeFromCheckout($checkoutData);
+
         if ($options['package_type'] === 'default') {
             $packageType = self::$defaultOptions->getPackageType();
         } else {
-            $packageType = (int)$options['package_type'] ?: 1;
+            $packageType = (int) $options['package_type'] ?: 1;
         }
 
-        if ($address->getCountryId() != 'BE' && (int)$options['package_type'] == 2) {
+        if ($address->getCountryId() != 'NL' &&
```

Comment:
Klopt die NL wel? Waarom is dat aangepast van BE naar NL?

Reply:
Done